core/src/main/java/dev/triumphteam/gui/guis/BaseGui.java

```diff
@@ -110,6 +110,9 @@ public abstract class BaseGui implements InventoryHolder {
     private boolean runCloseAction = true;
     private boolean runOpenAction = true;
 
+    // Is this a Folia server?
+    private boolean folia = false;
```

Comment:
Make the field static final, since it's only needed to check once and assign the method value to it directly, instead of putting it in the constructor.